tck/src/test/java/io/smallrye/opentelemetry/tck/arquillian/FixedResteasyServletInitializer.java

```diff
@@ -0,0 +1,146 @@
+package io.smallrye.opentelemetry.tck.arquillian;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.servlet.ServletContainerInitializer;
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRegistration;
+import javax.servlet.annotation.HandlesTypes;
+import javax.ws.rs.ApplicationPath;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Application;
+import javax.ws.rs.ext.Provider;
+
+import org.jboss.resteasy.core.AsynchronousDispatcher;
+import org.jboss.resteasy.plugins.server.servlet.HttpServlet30Dispatcher;
+import org.jboss.resteasy.plugins.server.servlet.ResteasyContextParameters;
+
+/**
+ * This is a fixed replacement for <a href=
+ * "https://github.com/resteasy/Resteasy/blob/master/resteasy-servlet-initializer/src/main/java/org/jboss/resteasy/plugins/servlet/ResteasyServletInitializer.java">ResteasyServletInitializer.java</a>.
+ * See also <a href="https://issues.jboss.org/browse/RESTEASY-1922">RESTEASY-1922</a>.
+ */
+@HandlesTypes({ Application.class, Path.class, Provider.class })
```

Comment:
Any reason this isn't in the implementation?

Reply:
This is a copy-paste from SmallRye-OpenTracing and if I remember correctly the fix was also applied to other TCKs https://github.com/search?q=org%3Asmallrye+FixedResteasyServletInitializer&type=Code

The referenced bug above has been fixed however the TCK fails without this fix. Maybe somebody from RestEasy could have a look?